chsdi/templates/htmlpopup/lubis.mako

```diff
@@ -233,7 +271,7 @@ viewer_url = get_viewer_url(request, params)
   </div>
   <br>
 
-% if preview_url != "" and c['attributes']['image_width']:
+% if preview_url != "" and image_width != None:
```

Comment:
yes, when the metafile does not exist (meaning lubis tile for this image not yet created, or in process of being created) or some exception thrown in the imagesize_from_metafile function.